7.application_configuration.md

```diff
@@ -33,7 +33,7 @@ In addition, as an application configuration is released, its _component instanc
 
 To accommodate this definition of a release, an OAM platform SHOULD make the following assumptions:
 
-- A component (as defined in the [component model](4.component.md) section) is immutable. It should not be changed after creation.
+- A component (as defined in the [component model](4.component.md) section) is mutable. Upgrade can happen by updating the component, e.g. updating image.
```

Comment:
Should merge this paragraph with the paragraph below, because change on a Component will results a new release.

Reply:
Sure. Will do it.